concrete/controllers/search/express/entries.php

```diff
@@ -27,28 +29,37 @@ public function search(Entity $entity)
         $this->entryList = new EntryList($entity);
         $this->entity = $entity;
 
-        $set = $this->entity->getResultColumnSet();
-        if ($set) {
-            $defaultSortColumn = $set->getDefaultSortColumn();
-            if ($this->request->query->has($this->entryList->getQuerySortDirectionParameter())) {
-                $direction = $this->request->query->get($this->entryList->getQuerySortDirectionParameter());
-            } else {
-                $direction = $defaultSortColumn->getColumnDefaultSortDirection();
-            }
-            if ($this->request->query->has($this->entryList->getQuerySortColumnParameter())) {
-                $value = $this->request->query->get($this->entryList->getQuerySortColumnParameter());
-                $column = $this->entity->getResultColumnSet();
-                $value = $column->getColumnByKey($value);
-                if (is_object($value)) {
-                    $this->entryList->sanitizedSortBy($value->getColumnKey(), $direction);
+        $app = Facade::getFacadeApplication();
```

Comment:
Since this is a controller you should have `$this->app`

Reply:
Sadly, I do not.